src/main/java/org/json/JSONArray.java

```diff
@@ -889,7 +889,7 @@ public String toString() {
         try {
             return this.toString(0);
         } catch (Exception e) {
-            return null;
+            return "";
```

Comment:
Why this change? `null != ""`

Reply:
Fixed in d3cd3f4c88c67b3621b6714e4af99b3048ad605d